src/composables/sdk.ts

```diff
@@ -30,11 +33,30 @@ export function useSdk({ store }: UseSdkOptions) {
     return watchUntilTruthy(() => sdk.value);
   }
 
+  /**
+   * Get the SDK instance. For now the SDK state is asynchronous.
+   * TODO: With the new SDK version this probably could be replaced with a computed prop.
+   */
```

Comment:
As the method allows to lazy load/instantiate the drySdk this comment makes no sense. Instead I suggest to add a comment why we need to have the `drySdk`.